rdrf/rdrf/templates/registration/password_reset_confirm.html

```diff
@@ -4,6 +4,13 @@
 
 {% block content %}
 
+<script type="text/javascript" src="{% static 'js/show_password.js' %}"></script>
```

Comment:
On this page, the password is pre-filled to `None` so it ends up looking like this:

![image](https://user-images.githubusercontent.com/17246877/135552964-2dde4ba2-7cf5-44b2-893b-50a1be7422ae.png)
![image](https://user-images.githubusercontent.com/17246877/135552976-97297fb9-6fc2-4f08-be41-aa45a0f86b8e.png)

It should probably be empty instead

Reply:
Ah yes, I did start looking at this but gave up as I was having issues getting this to work for me. I'll be in touch :)

Reply:
Looks like Pycharm was throwing me off by telling my syntax was invalid, but it does actually work! Ugh.

Reply:
Fixed.

Reply:
Also needs to be fixed in the password reset template

Reply:
Fixed the password reset template.